apps/public-docsite-v9/src/DocsComponents/ThemePicker.stories.tsx

```diff
@@ -1,5 +1,14 @@
 import * as React from 'react';
-import { MenuButton, Menu, MenuPopover, MenuItemRadio, MenuList, MenuTrigger, makeStyles, MenuProps } from '../index';
+import {
+  MenuButton,
+  Menu,
+  MenuPopover,
+  MenuItemRadio,
+  MenuList,
+  MenuTrigger,
+  makeStyles,
+  MenuProps,
+} from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import {
  makeStyles,
  Menu,
  MenuButton,
  MenuItemRadio,
  MenuList,
  MenuPopover,
  MenuProps,
  MenuTrigger,
} from '@fluentui/react-components';
```